clio-server/config/docker-compose.yml.ctmpl

```diff
@@ -10,8 +10,6 @@ app:
     tag: "clio-server"
   env_file:
     - {{$app_dir}}/{{$env_file}}
-  ports:
-    - "{{env "HOST_CLIO_PORT"}}:{{env "CONTAINER_CLIO_PORT"}}"
```

Comment:
Do we need `HOST_CLIO_PORT` as a variable anymore if we get rid of this? It's still used in the deploy script, but just so we can support curl-ing ports other than 80l